### DISCUSSION TITLE
Unable to Join Conference from Android Demo Application

### DISCUSSION THREAD

**Discussion by rushikakpatel1989** [ant-media/Ant-Media-Server#6539] [Q&A]
Hi Team,

I am using antmedia version 2.10.
I have license version, I am using Android sample app to connect conference but It is not working. It is showing connected and live but I can not see other participants. 
I have another setup but on that its working fine but on my license setup its not working.

On Android debug, I am getting below errors:

(line 608): UDP send of 20 bytes to host stun1.l.google.com:19302 ([2001:4860:4864:x:x:x:x:x]:19302) failed with error 0 : [0x00000001] Operation not permitted

2024-08-01 10:18:31.007 11483-14024 stun_port.cc    
        io....dia.webrtc_android_sample_app  E  (line 608): UDP send of 20 bytes to host stun1.l.google.com:19302 (74.125.250.x:19302) failed with error 0 : [0x00000016] Invalid argument
 
 
   io....dia.webrtc_android_sample_app  D  IceCandidateError address: 127.0.0.x, port: 42781, url: stun:stun1.l.google.com:19302, errorCode: 701, errorText: STUN binding request timed out.

Please guide me, is it configuration problem or issue is on antmedia version.

**Comment by yashtandon113** [comment]
Hi @rushikakpatel1989 

Thanks for reporting the issue.

I downloaded the Android SDK zip file from the [master branch](https://github.com/ant-media/WebRTC-Android-SDK) and tested the conference sample app with v2.10.0 of AMS and it seems to be working as expected. Below is my scenario:

1. Run the SDK on device
2. Join the room using the Conference sample app
3. Joined the same room using the sample web page conference.html

Please make sure that on your new server, all required ports are opened and SSL is enabled on the server.

Please check once again on your end and let us know if you find any such issue related to ports or SSL.

**Reply by rushikakpatel1989** [reply]
Can you tell me which are the port range we need to check. SSL is working fine, that I had confirmed.

**Reply by yashtandon113** [reply]
@rushikakpatel1989 
Sorry for the delay in response from my side. Is your issue resolved or does it still persist?

For WebRTC, port range 50000–60000 should be whitelisted. Is there any firewall in your network that might also cause the blocking of UDP traffic? Are their any differences between your working environment and a non-working one?
